model_compression_toolkit/quantizers_infrastructure/trainable_infrastructure/common/base_trainable_quantizer.py

```diff
@@ -27,6 +27,16 @@
     QUANTIZATION_TARGET
 
 
+class VariableGroup(Enum):
+    """
+    An enum for choosing training variable group
+    0. WEIGHTS
+    1. THRESHOLDS
+    """
+    WEIGHTS = 0
+    THRESHOLDS = 1
```

Comment:
QUANTIZATION_PARAMS  is too long and sounds like configuration instead of values.

Reply:
> QUANTIZATION_PARAMS is too long and sounds like configuration instead of values.

So we can think of a different name, but THRESHOLDS is not correct.
Anyway, quantization_params is what we call this parameters in other places of the project (we have "quantization_params_generation", that's where this parameters come from, that's why I think it's the right name)